Cargo.toml

```diff
@@ -200,6 +200,9 @@ substrate-prometheus-endpoint  = { git = "https://github.com/paritytech/substrat
 substrate-wasm-builder         = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
 try-runtime-cli                = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
 
+# Temproary Asset Registry
```

Comment:
Resolve conflict

Reply:
to be removed. New AssetRegistry already on development branch